gslib/commands/iam.py

```diff
@@ -499,7 +538,14 @@ def _PatchIam(self):
 
     # TODO: Add an error counter for files and objects.
     if not self.everything_set_okay:
-      raise CommandException('Some IAM policies could not be patched.')
+      msg = 'Some IAM policies could not be patched.'
+      if self.tried_ch_on_resource_with_conditions:
+        msg += '\n'
+        msg += '\n'.join(textwrap.wrap(
+           'Some resources had conditions present in their IAM policy '
```

Comment:
would it make sense to have same message here and above (line 427-429)?

Reply:
No -- one message is for a specific resource, while the other is a "summary" message, meaning that "one or more" resources ran into this problem.  The general pattern here (and in other places in gsutil where we support this continue-on-error mode) is that users may re-run the command without the continue-on-error flag, or run in debug mode (`-D`) to see which resources/requests are producing errors.

Reply:
Ack